File: app/src/main/java/com/tim/iot/device/local/ILocalServer.java
package com.tim.iot.device.local;

/**
 * ILocalServer
 *
 * @author Tell.Tim
 * @date 2019/12/27 11:09
 */
public interface ILocalServer {

    String getAccount();

    void saveAuthToLocal(String accountInfo);

    void clearAuthorized();
}

File: app/src/main/java/com/tim/iot/device/local/ILocalServer.kt
package com.tim.iot.device.local

/**
 * ILocalServer
 *
 * @author Tell.Tim
 * @date 2019/12/27 11:09
 */
interface ILocalServer {

    val account: String

    fun saveAuthToLocal(accountInfo: String)

    fun clearAuthorized()
}
